stdlib/2.7/typing.pyi

```diff
@@ -30,7 +30,7 @@ DefaultDict = TypeAlias(object)
 Set = TypeAlias(object)
 
 # Predefined type variables.
-AnyStr = TypeVar('AnyStr', str, unicode)
+AnyStr = TypeVar('AnyStr', bytes, str, unicode)
```

Comment:
This is key, but also annoying. E.g. this code breaks because of this:
```
def f(a): # type: (AnyStr) -> AnyStr
    if isinstance(a, str): return '.'
    else: return u'.'
```
You have to add a third case (`elif isinstance(a, bytes): return b'.'`). You can'y even collapse this as `isinstance(a, (str, bytes))` because the return type (in this example) must match the argument.